docs/components/modeler/bpmn/error-events/error-events.md

```diff
@@ -52,8 +51,8 @@ instance.
 
 It is not possible to define multiple error catch events with the same `errorCode` in a single scope. It is also not
 permitted to have multiple error catch events without an `errorCode` in a single scope. The deployment gets rejected in
-these cases. However, it is possible to define both an error catch event **with** an `errorCode` and one **without** an
-`errorCode` in the same scope. When this happens, the error catch event that matches the `errorCode` is prioritized.
+these cases. However, it is possible to define both an error catch event referencing an error with a particular `errorCode` and an error catch-all event within the same scope. When this happens, the error catch event
+that matches the `errorCode` is prioritized.
```

Comment:
👍 I like the changes, but I think there's another sentence that needs to be adjusted

>It is also not permitted to have multiple error catch events without an `errorCode` in a single scope.

🔧 Is that sentence still relevant? I believe it should say something like:

```
It is also not permitted to have multiple error catch-all events in a single scope.
```

Reply:
I've tried to incorporate this into https://github.com/camunda/camunda-platform-docs/pull/2471/commits/0cea47e66891bcc1389575cefd12018a2472c087.